jessica_hart/Assignments/Rails Controllers Views/bootstrap_layouts/app/views/layouts/three_column.html.erb

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title><%= yield :title %></title>
+    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
+    <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+    <%= csrf_meta_tags %>
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="col-md-4"><%= yield :col1 %></div>
```

Comment:
Cool use of content for, but this forces a two column layout everywhere. You can render specific layouts if you like, rather than forcing all into this one. http://guides.rubyonrails.org/layouts_and_rendering.html#finding-layouts